evap/settings.py

```diff
@@ -121,7 +121,16 @@
             'CLIENT_CLASS': 'django_redis.client.DefaultClient',
             'MAX_ENTRIES': 100000
         }
-    }
+    },
+    'sessions': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': 'redis://127.0.0.1:6379/2',
+        'TIMEOUT': None,  # is always invalidated manually
```

Comment:
specifying `TIMEOUT` doesn't make sense here. we don't want to cache anything forever in this cache, and the timeout is probably set by django according to the session timeout.